app/services/land-service.js

```diff
@@ -0,0 +1,21 @@
+const config = require('../config')
+const restClient = require('../utils/rest-client')
+
+const getRpaLandServiceUrl = function (entity, sbi) {
+  return `${config.rpaLandServiceUrl}/${entity}/MapServer/0/query?where=SBI=${sbi}&outFields=*&f=geojson`
+}
+
+const landService = {
+  async getParcels (sbi) {
+    const landParcelsUrl = getRpaLandServiceUrl('LandParcels', sbi)
+
+    const response = await restClient.get(landParcelsUrl)
```

Comment:
No error checking of the result, which results in the service failing if invalid data is supplied. Not a must fix for this quick throw together, but maybe add a TODO comment to fix this later.

Reply:
Good point. I've added handling for errors calling the API, unexpected response format and responses with no features.

The RPA API doesn't seem to throw errors for non-existent SBIs, it just returns the same data structure with an empty `features` array. Based on that, I've treated an empty `features` array as if it was a "not found" error so we can show some error handling through our front-end.